lua/terrortown/lang/en.lua

```diff
@@ -1737,46 +1738,46 @@ L.label_prop_carrying_force = "Prop pickup force"
 L.label_teleport_telefrags = "Kill blocking player(s) when teleporting (telefrag)"
 L.label_allow_discomb_jump = "Allow disco jump for grenade thrower"
 L.label_spawn_wave_interval = "Spawn wave interval in seconds"
-L.label_voice_enable = "Enable voicechat"
-L.label_voice_drain = "Enable the voicechat battery feature"
+L.label_voice_enable = "Enable voice chat"
+L.label_voice_drain = "Enable the voice chat battery feature"
 L.label_voice_drain_normal = "Drain per tick for normal players"
 L.label_voice_drain_admin = "Drain per tick for admins and public policing roles"
-L.label_voice_drain_recharge = "Recharge rate per tick of not voicechatting"
-L.label_locational_voice = "Enable locational 3D voicechat sound for living players"
+L.label_voice_drain_recharge = "Recharge rate per tick of not voice chatting"
+L.label_locational_voice = "Enable local 3D voice chat sound for living players"
```

Comment:
```suggestion
L.label_locational_voice = "Enable proximity voice chat for living players"
```
If this is changed further above, we should change it here as well

Reply:
Done. 👍